x/bank/keeper/keeper_test.go

```diff
@@ -0,0 +1,189 @@
+		err := sdksimapp.FundAccount(
+			suite.app.BankKeeper,
+			suite.ctx,
+			acc,
+			sdk.NewCoins(
+				sdk.NewCoin("uctk", sdk.NewInt(1000)), // 1,000 CTK
+			),
+		)
+		if err != nil {
+			panic(err)
+		}
+	}
+
+	suite.address = []sdk.AccAddress{acc1, acc2, acc3, acc4}
+}
+
+func TestKeeperTestSuite(t *testing.T) {
+	suite.Run(t, new(KeeperTestSuite))
+}
+
+func (suite *KeeperTestSuite) TestMsgSend() {
+	type args struct {
+		fromAddr sdk.AccAddress
+		toAddr   sdk.AccAddress
+		amount   int64
+	}
+
+	type errArgs struct {
+		shouldPass bool
+		contains   string
+	}
+
+	tests := []struct {
+		name    string
+		args    args
+		errArgs errArgs
+	}{
+		{"Operator(1) Create: first send",
+			args{
+				amount:   200,
+				fromAddr: suite.address[0],
+				toAddr:   suite.address[1],
+			},
+			errArgs{
+				shouldPass: true,
+				contains:   "",
+			},
+		},
+		{"Operator(1) Create: second send if balance is less",
+			args{
+				amount:   11000,
+				fromAddr: suite.address[0],
+				toAddr:   suite.address[1],
+			},
+			errArgs{
+				shouldPass: false,
+				contains:   "",
+			},
+		},
+	}
+	for _, tc := range tests {
+		suite.Run(tc.name, func() {
+			suite.SetupTest()
+			err := suite.app.BankKeeper.SendCoins(suite.ctx, tc.args.fromAddr, tc.args.toAddr, sdk.Coins{sdk.NewInt64Coin("uctk", tc.args.amount)})
+			balance := suite.app.BankKeeper.GetAllBalances(suite.ctx, tc.args.fromAddr)
+			if tc.errArgs.shouldPass {
+				suite.Require().NoError(err, tc.name)
+				suite.Require().NotEqual(tc.args.amount, balance)
+				suite.Require().Equal(sdk.Coins{sdk.NewInt64Coin("uctk", 800)}, balance)
+			} else {
+				suite.Require().Error(err, tc.name)
+				suite.Require().NotEqual(tc.args.amount, balance)
+				suite.Require().True(strings.Contains(err.Error(), tc.errArgs.contains))
+			}
+		})
+	}
+}
+
+func (suite *KeeperTestSuite) TestMsgMultiSend() {
+	type args struct {
+		fromAddr        sdk.AccAddress
+		toAddr          sdk.AccAddress
+		UnlockerAddress sdk.AccAddress
+		amount          int64
+	}
+
+	type errArgs struct {
+		shouldPass bool
+		contains   string
+	}
+
+	tests := []struct {
+		name    string
+		args    args
+		errArgs errArgs
+	}{
+
+		{"Operator(1) Create: first send",
+			args{
+				amount:          200,
+				fromAddr:        suite.address[0],
+				toAddr:          suite.address[1],
+				UnlockerAddress: suite.address[2],
+			},
+			errArgs{
+				shouldPass: true,
+				contains:   "",
+			},
+		},
+
+		{"Operator(1) Create: second send",
+			args{
+				amount:          1100,
+				fromAddr:        suite.address[0],
+				toAddr:          suite.address[1],
+				UnlockerAddress: suite.address[2],
+			},
+			errArgs{
+				shouldPass: false,
+				contains:   "",
+			},
+		},
+	}
+	for _, tc := range tests {
+		suite.Run(tc.name, func() {
+			suite.SetupTest()
+			err := suite.app.BankKeeper.SendCoins(suite.ctx, tc.args.fromAddr, tc.args.toAddr, sdk.Coins{sdk.NewInt64Coin("uctk", tc.args.amount)})
```

Comment:
I think you meant to test `LockedSend` here?

Reply:
yes